pkg/cluster/arm_test.go

```diff
@@ -115,6 +115,8 @@ func TestDeployARMTemplate(t *testing.T) {
 
 			err := m.deployARMTemplate(ctx, resourceGroup, deploymentName, armTemplate, params)
 
+			t.Fail()
```

Comment:
This is here to demonstrate the reduced test report output. Will not be part of the code.